.github/workflows/process-results.yml

```diff
@@ -7,26 +7,40 @@ on:
       extension:
         type: string
         default: "jsonl"
+      repository:
+        type: string
+        required: true
+    secrets:
+      gcp_credentials:
+        description: service account used to upload results to GCS
+        required: true
 
 jobs:
   process-results:
     runs-on: ubuntu-latest
     env:
       NAME: ${{ inputs.name }}
       EXTENSION: ${{ inputs.extension }}
+      REPOSITORY: ${{ inputs.repository }}
     steps:
       - uses: actions/checkout@v3
+      - id: 'auth'
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: '${{ secrets.gcp_credentials }}'
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+        with:
+          version: '>= 363.0.0'
       - uses: actions/download-artifact@v3
         with:
           name: latest-result
           path: ./
-      - name: Replace with the latest result
-        run: |
-          rm -rf results/$NAME/latest.$EXTENSION
-          mv latest.$EXTENSION results/$NAME
-      - name: Process results
+      - name: Upload results to GCS
         run: |
           FILENAME=$(date +%Y-%m-%d)
-          cd results/$NAME
           cp latest.$EXTENSION $FILENAME.$EXTENSION
           gzip $FILENAME.$EXTENSION
+
+          gsutil -m cp -r $FILENAME.$EXTENSION.gz gs://egq-runziggurat-$REPOSITORY-bucket/results/$NAME
```

Comment:
Could we also upload/overwrite the `latest.$FILENAME` file here? It's used in a couple of other steps and probably in the GUI as well.

Reply:
you mean `latest.$EXTENSION` ? I think we can also do that as well, so the compressed and uncompressed file?

Reply:
Yeah, it would be nice to avoid having to find latest by file name, and then also decompress it

Reply:
sure! I'll open a PR for uploading the `latest.$EXTENSION` as well!

Reply:
Awesome, thanks!